pinot-common/src/main/java/org/apache/pinot/common/metadata/ZKMetadataProvider.java

```diff
@@ -98,6 +99,10 @@ public static InstanceZKMetadata getInstanceZKMetadata(ZkHelixPropertyStore<ZNRe
     return new InstanceZKMetadata(znRecord);
   }
 
+  public static String constructPropertyStorePathForTask(String resourceName, String taskId) {
```

Comment:
Suggest not keeping per-task level ZK metadata. It might create too many ZNode, and can be easily left over if not handled properly